### PR TITLE
curl.1: add an "OUTPUT" section at the top of the manpage

### PR DESCRIPTION
**Pull request by bagder**
Explain the basic concepts behind curl output.

Inspired by #6124

/cc @jzakrzewski and @jjatria

### CODE REVIEW

File: docs/cmdline-opts/page-header [hunk 1]
```diff
@@ -99,6 +99,15 @@ getting many files from the same server will not do multiple connects /
 handshakes. This improves speed. Of course this is only done on files
 specified on a single command line and cannot be used between separate curl
 invokes.
+.SH OUTPUT
+If not told otherwise, curl writes output to stdout. It can be instructed to
```
Comment (emilengler) [inline docs/cmdline-opts/page-header:103]: This is wrong. Verbose and errors are written to stderr

**Review by emilengler — CHANGES_REQUESTED** [review]

File: docs/cmdline-opts/page-header (hunk 1 above)
Comment (bagder) [reply docs/cmdline-opts/page-header:103]: In my view that's not "output" but from your reaction I understand I should probably qualify that in the text...

Comment (emilengler) [reply docs/cmdline-opts/page-header:103]:
```
curl --help category | grep verbose
 verbose     Options related to any kind of command line output of curl
```
It is output :P

Comment (bagder) [reply docs/cmdline-opts/page-header:103]: Again, that's not matching my personal terminology. Verbose output is not plain "output".

Comment (bagder) [reply docs/cmdline-opts/page-header:103]: To me, the output of curl is the data it outputs.

Comment (emilengler) [reply docs/cmdline-opts/page-header:103]: Ok your decision then. In my personal opinion output means everything visible on the shell. Of course this isn't standardized somehow but every average shell displays stdout and stderr.

**Pull request closed without merging**